zhaquirks/xiaomi/__init__.py

```diff
@@ -449,6 +448,10 @@ class XiaomiAqaraE1Cluster(XiaomiCluster, ManufacturerSpecificCluster):
     cluster_id = 0xFCC0
 
 
+class OppleCluster(XiaomiAqaraE1Cluster):
+    ep_attribute = "opple_cluster"
+
+
```

Comment:
When https://github.com/zigpy/zha-device-handlers/pull/2667 is merged, this can be removed (or now already).
The `ep_attribute = "opple_cluster"` technically isn't needed for this PR, since it's not accessed by ZHA or tests u sing the endpoint attribute.

Reply:
Done